internal/pkg/helm/values_inspector.go

```diff
@@ -91,3 +93,63 @@ func isBooleanYamlValue(input *map[string]interface{}, path *[]string, pathIndex
 		}
 	}
 }
+
+
+//GetValues .. Get values  fro given path
+func GetValues(arg string) (*map[string]interface{},  error) {
+
+	argString := strings.Replace(arg, ".Values.", "", -1)
+	chartClient := NewChartClient()
+	err := chartClient.LoadChartFrom(HelmChartRef)
+	if err != nil {
+		logrus.Warnf("error loading chart: %s", err)
+		return nil, err
+	}
+	raw, _ := chartutil.ReadValues([]byte(chartClient.Chart.Values.Raw))
+	result,err := raw.PathValue(argString)
+	if err!=nil{
+		return nil, err
+	}
+	return dump(result), nil
+}
+
+//need some improvements  here
+func dump(result interface{}) *map[string]interface{} {
+	items := reflect.ValueOf(result)
+	if items.Kind() == reflect.Slice {
+		//for i := 0; i < items.Len(); i++ { // just get first level for now
+		if items.Len()>0{
+			item := items.Index(0)
+			returnMap:= item.Interface().(map[string]interface{})
+			return &returnMap
+		}
+	}
+	return nil
+}
+func PreFixValuesWithItems(sb *strings.Builder, itemField string ,values *map[string]interface{}){
+	input:=sb.String()
+	lines := strings.Split(string(input), "\n")
+	lineNumbers := []int{}
+	itemField=replaceDollar + itemField +"." //Issue with $ sign signs are interpreted as in Expand
```

Comment:
What does this comment mean?  Also, it appears that "." is really the constant "goTemplateMemberAccessOperator".

Reply:
It has been removed. Outdated now